packages/cli/src/constructs/retry-strategy.ts

```diff
@@ -55,6 +55,13 @@ export class RetryStrategyBuilder {
     return RetryStrategyBuilder.retryStrategy('EXPONENTIAL', options)
   }
 
+  /**
+   * No retries are performed.
+   */
+  static noRetries (): null {
```

Comment:
I couldn't think of a good name that matches the `<something>Strategy` pattern that we have.